.drone.yml

```diff
@@ -425,14 +425,14 @@ steps:
     pull: if-not-exists
     image: plugins/slack
     settings:
-      channel: sas-build
+      channel: sas-hof-build-notify
       failure: ignore
       icon_url: http://readme.drone.io/0.5/logo_dark.svg
```

Comment:
that url returns 404.
as agreed to use this drone icon instead
`https://readme.drone.io/logo.svg`

Reply:
icon.url appears twice

Reply:
Thanks @dk4g . I have corrected the image url